src/components/pages/comparison/map-components/DataDisplay.js

```diff
@@ -173,11 +173,80 @@ const DataDisplay = ({
+                  <div className={`menu-items ${menu.status}`}>
+                    <div className="data-nav-top">
+                      <MapSearch
+                        menu={menu.status}
+                        setSearch={setSearch}
+                        onSearch={onSearch}
+                        cityMarkers={cityMarkers}
+                        search={search}
+                        viewport={viewport}
+                        setViewport={setViewport}
+                        selectSearch={selectSearch}
+                        cityIndex={cityIndex}
+                      />
+                      {selected.length > 0 ? (
+                        <div className="anchor-nav">
+                          <ul>
+                            {selected.map((item) => (
+                              <div
+                                key={item._id}
+                                className={`menu-items ${menu.status}`}
+                              >
+                                <li key={item._id}>
+                                  <span className="color-legend-text">
+                                    <div
+                                      className="color-legend"
+                                      style={{
+                                        display: "inline-block",
+                                        background: item.color,
+                                        height: "1rem",
+                                        width: "1rem",
+                                        marginRight: ".5rem",
+                                      }}
+                                    ></div>
+                                    {item.name_with_com}
+                                  </span>
+                                  <span
+                                    onClick={(_) => {
+                                      let foundCity = cityIndex.find(
+                                        (indexed) => indexed.ID === item._id
+                                      );
+                                      toggleSelected(foundCity);
+                                    }}
+                                  >
+                                    <img
+                                      className="delete-icon"
+                                      src={deleteIcon}
+                                      alt="delete icon"
+                                    />
+                                  </span>
+                                </li>
+                              </div>
+                            ))}
+                          </ul>
+                        </div>
+                      ) : null}
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </nav>
+          </div>
+       
+      </div>
       {/* <div className="nav-box"> */}
-      <div className="nav-placeholder">
+      {/* <div className="nav-placeholder">
```

Comment:
You should remove commented out code. A clean code base is a speedy codebase. 

I'd argue if it's not good enough to not be commented out, it doesn't belong in master. 

If you're still debating whether or not you need this code, you can stash the commented out stuff onto another branch and come back to it :)